locustfile.py

```diff
@@ -51,47 +436,26 @@ def login(self):
             if post_response.url != HOST + redirect_url:
```

Comment:
Authentication was "breaking" for me because of the aforementioned hardcoded host issue. Maybe we can just check `if post_response.url.endswith(redirect_url): ...`?

Reply:
I changed `HOST` to `self.locust.host` and it seems happy.